cmake/modules/SwiftSource.cmake

```diff
@@ -317,19 +317,32 @@ function(_compile_swift_files
     list(APPEND swift_flags "-parse-as-library")
 
     set(module_base "${module_dir}/${SWIFTFILE_MODULE_NAME}")
+    if(SWIFTFILE_SDK IN_LIST SWIFT_APPLE_PLATFORMS)
+      set(specific_module_dir "${module_base}.swiftmodule")
```

Comment:
You only use this once below, I think you could just inline it there as it is already covered by the conditional.

Reply:
It's used to form `module_base` as well as being passed to the `install` command. I didn't want the `install` to have to know about the particular extension here.